server.js

```diff
@@ -7,6 +7,10 @@ global.config = config
 const logger = require('./utils/logger')
 global.logger = logger
 
+logger.info(`Initialising newrelic with app name:: ${config.get('integrations.newrelic.appName')}`)
+// Initialise newrelic
+require('newrelic')
+
```

Comment:
These changes already exist, can we remove them from this PR?

Reply:
Already existing changes removed from this PR